src/exampleTypes/ResultTable/index.js

```diff
@@ -62,34 +65,38 @@ let ResultTable = ({
     criteria,
   }
 
-  return (
-    <>
-      <Table>
-        <thead>
-          <tr>
-            {F.mapIndexed(
-              x => (
-                <Header key={x.field} field={x} {...headerProps} />
-              ),
-              visibleFields
-            )}
-            <HighlightedColumnHeader node={node} />
-          </tr>
-        </thead>
-        <TableBody
-          {...{
-            node,
-            fields,
-            visibleFields,
-            hiddenFields,
-            schema,
-            Row,
-          }}
-        />
-      </Table>
-      <ResultTableFooter {...{ tree, node, path, pageSizeOptions }} />
-    </>
-  )
+  if (hasResults) {
```

Comment:
Nitpick: if you do `if (!hasResults) return ...` you can save indentation on the component